File: ce/customer-engagement/on-premises/snippets/visualbasic/CRMV8/businessmanagementvb/vb/createconnection1.vb
' <snippetcreateconnection1>


 ' Create a connection between the account and the contact.
 ' Assign a connection role to a record.
 Dim newConnection As Connection =
     New Connection With
     {
         .Record1Id =
         New EntityReference(Account.EntityLogicalName,
                                          _accountId),
         .Record1RoleId =
         New EntityReference(ConnectionRole.EntityLogicalName,
                             _connectionRoleId),
         .Record2RoleId =
         New EntityReference(ConnectionRole.EntityLogicalName,
                             _connectionRoleId),
         .Record2Id =
         New EntityReference(Contact.EntityLogicalName,
                                          _contactId)
     }

 _connectionId = _serviceProxy.Create(newConnection)

' </snippetcreateconnection1>

File: ce/customerengagement/on-premises/snippets/csharp/CRMV8/businessmanagement/cs/createconnection1.cs
// <snippetcreateconnection1>


 // Create a connection between the account and the contact.
 // Assign a connection role to a record.
 Connection newConnection = new Connection
 {
     Record1Id = new EntityReference(Account.EntityLogicalName,
         _accountId),
     Record1RoleId = new EntityReference(ConnectionRole.EntityLogicalName,
         _connectionRoleId),                             
     Record2RoleId = new EntityReference(ConnectionRole.EntityLogicalName,
         _connectionRoleId),                            
     Record2Id = new EntityReference(Contact.EntityLogicalName,
         _contactId)
 };

 _connectionId = _serviceProxy.Create(newConnection);

// </snippetcreateconnection1>